frontend/pages/projects/_id/article-annotation/index.vue

```diff
@@ -408,6 +425,45 @@ export default {
     },
     changeSelectedLabel(event) {
       this.selectedLabelIndex = this.spanTypes.findIndex((item) => item.suffixKey === event.srcKey)
+    },
+
+    removeTag(annotationId) {
+      // await this.$services.seq2seq.delete(this.projectId, this.doc.id, id)
+      // await this.list(this.doc.id)
+      for (let i = 0; i < this.affectiveSummaryTags.length; i++) {
+        if (this.affectiveSummaryTags[i].id === annotationId) {
+          this.affectiveSummaryTags.splice(i, 1)
+          break
+        }
+      }
+    },
+    updateTag(annotationId, text) {
+      console.log(annotationId, text)
+    },
+    addTag(value) {
+      const item = {
+        "id": this.affectiveSummaryTags.length,
+        "text": value
+      }
+      this.affectiveSummaryTags.push(item)
+    },
+    removeImpression(annotationId) {
+      for (let i = 0; i < this.affectiveSummaryImpressions.length; i++) {
```

Comment:
I think this can be simplified by using the `Array.find` function, what do you think?

Reply:
Most probably yes, I will change it too XD